pkg/table/table.go

```diff
@@ -378,6 +421,13 @@ func (m *Model) updateColumnWidths() {
 	m.columnWidths = lengths
 }
 
+func (m *Model) filterText() string {
+	if !m.filterInputEnabled {
+		return ""
+	}
+	return m.filterInput.Value()
+}
+
```

Comment:
Would be nice if you could leave the filter input as-is.

That way, you can close the filter field, and then use other hotkeys.

Maybe also add a "filtering 12/30 rows" to the gray status bar at the bottom